_includes/configuration/style-formats.md

```diff
@@ -3,9 +3,10 @@
 
 This option allows you to define custom items for the `styleselect` dropdown toolbar button and the `formats` menu item.
 
-There are three types of items the array can contain:
+There are four types of items the array can contain:
 
-* **Style:** The item must have a `title` and either a `format` with refers to a pre-registered editor [format]({{ site.baseurl }}/configure/content-formatting/#formats) or the necessary fields to [specify a new format]({{ site.baseurl }}/configure/content-formatting/#formattype). The item will be rendered in the UI as a clickable item that applies the given format.
+* **Style:** The item must have a `title` and an optional `name` to specify what it will be registered as. Then it needs the other necessary fields to [specify a new format]({{ site.baseurl }}/configure/content-formatting/#formattype). If the name is specified it will be prefixed with `custom-` this is to ensure that it doesn't clash with the default editor formats. The item will be rendered in the UI as a clickable item that applies the given format.
```

Comment:
Can we change the third sentence:

> If the name is specified it will be prefixed with `custom-` this is to ensure that it doesn't clash with the default editor formats.

It's not really clear from this what the prefixed name is used for, or will happen if the name field is not specified. Can we add more detail here, or is this something that readers would know from elsewhere in this area of the docs?

Reply:
Added some more details. However maybe this entire page should be restructured I don't like it when a paragraph becomes a things that talks about many different things. Better to use some form of a list with all fields and sub sections for each type. But not sure if tyler agrees also a bit of work to do that.